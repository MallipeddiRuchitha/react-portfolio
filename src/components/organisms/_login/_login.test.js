import React from 'react';
import { shallow ,mount} from 'enzyme';
import Login from './_login';
import AuthContext from '../../../AuthContext';
const auth={
    login:jest.fn()
}
describe("Login component",() =>{
    test("Matches the snapshot",() => {
        const login = shallow( <AuthContext.Provider value={auth}>< Login /></AuthContext.Provider>);
     expect(login).toMatchSnapshot();
        

    })
    test("Login element is rendered",() => {
        const wrapper = mount(<AuthContext.Provider value={auth}>< Login /></AuthContext.Provider>);
        expect(wrapper.find('#login').exists()).toEqual(true);
        wrapper.unmount();
        
    });
   
   
   

});