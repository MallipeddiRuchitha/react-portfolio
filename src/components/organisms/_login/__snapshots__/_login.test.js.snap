// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component Matches the snapshot 1`] = `<Login />`;
